packages/loopring_v3/contracts/impl/libexchange/ExchangeData.sol

```diff
@@ -164,6 +164,9 @@ library ExchangeData
     function GENESIS_MERKLE_ROOT() internal pure returns (bytes32) {
         return 0x2b4827daf74c0ab30deb68b1c337dec40579bb3ff45ce9478288e1a2b83a3a01;
     }
+    function SNARK_SCALAR_FIELD() internal pure returns (uint256) {
+        return 21888242871839275222246405745257275088548364400416034343698204186575808495617;
```

Comment:
I wonder how this number is generated? its hex value is 0x30644e72e131a000000000000000000000000000000000000000000000000000

Reply:
It's the prime number that is used for the field for the BN128 curve. See [EIP-196](http://eips.ethereum.org/EIPS/eip-196) for example.

Reply:
i see, can you add your response as a comment to the code.